### PR TITLE
Login redirects

### LINKED ISSUE #43: fix spelling (login page)
Password -> Passwort

### PR DESCRIPTION
**Pull request by AlgorithmicEntropy**
The login page can now redirect after a successful login to a predefined page.
The redirect url needs to be stored as a browser cookie under the 'redirect' tag

### CODE REVIEW

File: page/login.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ function page (req, res) {
   var fs = require('fs')
   fs.readFile('html/login.html', 'utf8', function (err, html) {
     if (err) throw err
-    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login'] }))
+
+    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login', 'cookie_parser'] }))
```
Comment (DavidRisch) [inline page/login.js:12]: No js file with the name `cookie_parser` exists, this results in a 404.

File: page/login.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ function page (req, res) {
...
+
```
Comment (DavidRisch) [inline page/login.js]: Please use the formatter: `standard --fix`

**Review by DavidRisch — CHANGES_REQUESTED** [review]

File: page/login.js (hunk 2 above)
Comment (AlgorithmicEntropy) [reply page/login.js]: Empty line was accidentally added by me when fixing conflicts in github, fixed by latest commit

File: page/login.js (hunk 1 above)
Comment (AlgorithmicEntropy) [reply page/login.js:12]: Fixed by latest commit

**Comment by AlgorithmicEntropy** [comment]
Fixed spelling, closes #43

**Pull request merged**